_docs/reference/config/mixer/attribute-vocabulary.md

```diff
@@ -63,4 +63,8 @@ deployments will have agents (Envoy or Mixer adapters) that produce these attrib
 | connection.sent.bytes_total | int64 | Total number of bytes sent by a destination service during the lifetime of a connection. | |
 | connection.duration | duration | The total amount of time a connection has been open. | |
 | context.protocol | string | Protocol of the request or connection being proxied. | tcp |
-| context.time | timestamp | The timestamp of the Report() call. | |
\ No newline at end of file
+| context.time | timestamp | The timestamp of Mixer operation. | |
+| api.service | string | The public service name. One API could be handled by multiple internal services. Or multiple APIs could be handled by a single internal service. | my-svc.com |
+| api.version | string | The API version. | v1alpha1 |
+| api.operation | string | Unique string used to identify the operation. The id MUST be unique among all operations described in a specific <service, version>. | getPetsById |
```

Comment:
I actually don't understand the comment about MUST be unique. An attribute is basically a parameter to a function. There is no uniqueness requirement on this parameter. There might be a uniqueness requirement when the operation is declared in an API spec, but its not relevant to this document.

So I'd remove the whole second sentence.

Reply:
The policies are defined based on the operation. This uniqueness documentation helps clarify the assumptions.

Reply:
I still think you could remove that. But if you insist on keeping the sentence, then MUST should be changed to 'is'.